cardano-testnet/test/cardano-testnet-golden/files/golden/help/cardano.cli

```diff
@@ -14,6 +14,7 @@ Usage: cardano-testnet cardano [--num-pool-nodes COUNT]
   [--enable-p2p BOOL]
   [--nodeLoggingFormat LOGGING_FORMAT]
   [--num-dreps NUMBER]
+  [--enable-new-epoch-state-logging]
```

Comment:
I don't like this approach of slapping on additional args to the executable. However, I don't have anything else in mind at the moment. I think any parameters that exists in the genesis files or node config files should not appear as a command line option because this introduces the problem of precedence.

Reply:
I agree. If you feel that this option is unnecessary in `cardano-testnet` command line interface, I can hardcode its value to `False`. What do you think?